packages/cozy-pouch-link/src/jsonapi.js

```diff
@@ -20,7 +20,9 @@ export const normalizeDoc = (doc, doctype) => {
 
 export const fromPouchResult = (res, withRows, doctype) => {
   if (withRows) {
-    const docs = res.rows ? res.rows.map(row => row.doc) : res.docs
+    const docs = res.rows
+      ? res.rows.map(row => row.doc).filter(Boolean)
```

Comment:
I'm wondering if we will remember what this line is in a few months. Maybe we can create a method with an explicit name?